wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre software for musicians to perform, rehearse or just jam — all in real time over the Internet."
 mAltProgIcon: "Jamulus icon"
-mTSlogan: "Play music online. With friends. For free."
+mTSlogan: "Play music online. With friends."
 mTGetStartedNow: "Get started now!"
 mTDownloadNow: 'Download now for'
 mTOtherPlatforms: 'other platforms'
 ---
 <div class="fx-row fx-row-center-xs" id="firstrow">
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
-      <h2>What is Jamulus?</h2>
-       Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. With high quality, low-latency sound on a normal broadband connection, it's easy to play together remotely and in time. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
+      <h2>Jamulus?</h2>
+      <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Set it up now</a> and play together remotely. Collaborative high quality, low-latency sound via broadband. With friends, your band, or someone found online.
```

Comment:
We already have a very prominent Getting Started button. I do not think that we need it here again. Give a short description which includes the most important key phrases and use cases. This statement should inform users and Google about what the app is about.

Reply:
There are two buttons. One is positioned before any of the info, so it is a tough sell.
The other is all the way at the bottom, possibly out of view.
Neither of them is "Getting started", but rather instructions to the contrary.